sources/dynamodb/streaming.go

```diff
@@ -407,9 +407,9 @@ func parentDataMissingError(err error) bool {
 }
 
 // writeMutation handles writing of a mutation to Cloud Spanner. If insertion fails
-// because of parent data missing error then it retries for a limit of 1000.
+// because of parent data missing error then it retries for a limit of 100.
 func writeMutation(m *sp.Mutation, streamInfo *StreamingInfo) error {
-	retryLimit := 1000
+	retryLimit := 100
```

Comment:
Don't we have a constants or config file in harbourbridge where we can declare these constants?
If not, lets declare `retryLimit` as a constant instead of a variable.

Reply:
we have a constant file but this retryLimit is very specific to DynamoDB. So it wouldn't make sense to put in constants file.

Okay! will change retryLimit to a constant.